### ISSUE TITLE
Tiled processing

### ISSUE THREAD

**Issue by satatchan** [kijai/ComfyUI-GIMM-VFI#7]
It works pretty good on splitted images. I successfully managed to split input 4k sequence to 9 slices, interpolate each of them and then combine together. No visible seams so far. 
Otherwise there is always OOM error. Even with fullhd.

**Comment by yobada** [comment]
Can you please give me a workflow/some tips for nodes that will accomplish that? I am trying to do the same thing but running into issues.

**Comment by yobada** [comment]
Very cool! Much appreciated, I will test this out once I am home.

**Comment by scorpes** [comment]
Yes will there be an automated Comfyui workflow for this, so you can use this with Tiles?

**Comment by holycowdude** [comment]
Please can someone share a workflow which can do this? I'd love to try it out! 😊

**Comment by holycowdude** [comment]
Not yet, I need help to understand how to put the tensorsplit_combine.txt into the workflow please 😊
